### PR TITLE
yewtube 2.10.5

### PR DESCRIPTION
**Pull request by branchv**
mps-youtube: migrate to `python@3.12`

### CODE REVIEW

File: Formula/m/mps-youtube.rb [hunk 1]
```diff
@@ -22,20 +22,15 @@ class MpsYoutube < Formula
...
-  resource "youtube-dl" do
-    url "https://files.pythonhosted.org/packages/51/80/d3938814a40163d3598f8a1ced6abd02d591d9bb38e66b3229aebe1e2cd0/youtube_dl-2020.5.3.tar.gz"
-    sha256 "e7a400a61e35b7cb010296864953c992122db4b0d6c9c6e2630f3e0b9a655043"
-  end
-
   def install
-    venv = virtualenv_create(libexec, "python3.11")
+    venv = virtualenv_create(libexec, "python3.12")
 
     %w[youtube-dl pafy].each do |r|
```
Comment (onlined) [inline Formula/m/mps-youtube.rb:35]:
```suggestion
    %w[youtube-dl pafy].each do |r|
```

Comment (onlined) [inline Formula/m/mps-youtube.rb:35]: You forgot to remove `youtube-dl` from here.

Comment (iMichka) [reply Formula/m/mps-youtube.rb]: Done

**Comment by branchvincent** [comment]
Last release on pypi was Feb 2018, looks like the project was forked and renamed to `yewtube` (which https://github.com/mps-youtube/mps-youtube also redirects to). I'll update this PR to be a rename instead

File: Formula/y/yewtube.rb [hunk 2]
```diff
@@ -0,0 +1,126 @@
+class Yewtube < Formula
+  include Language::Python::Virtualenv
+
+  desc "Terminal based YouTube player and downloader"
+  homepage "https://github.com/mps-youtube/yewtube"
+  url "https://github.com/mps-youtube/yewtube/archive/refs/tags/v2.10.5.tar.gz"
+  sha256 "8100466a5e89c84249d882e2e9ea9ff282a2d4f7f68135157cb942e6eb927b29"
+  license "GPL-3.0-or-later"
+
+  depends_on "mplayer"
```
Comment (SMillerDev) [inline Formula/y/yewtube.rb:10]:
```suggestion
  depends_on "ffmpeg"
  depends_on "mplayer"
```

**Comment by branchvincent** [comment]
only an audit failure: 

`Formulae should not require patches to build. Patches should be submitted and accepted upstream first.`

**Review by fxcoudert — APPROVED** [review]

**Pull request merged**